helix-term/src/commands.rs

```diff
@@ -4570,13 +4574,18 @@ fn shell_impl(
             return Err(e.into());
         }
     };
-    if let Some(input) = input {
-        let mut stdin = process.stdin.take().unwrap();
-        for chunk in input.chunks() {
-            stdin.write_all(chunk.as_bytes())?;
+    let mut stdin = process.stdin.take().unwrap();
```

Comment:
This can panic when `input.is_none()`, for example `:sh sleep 1`

Reply:
Oh, I took this straight from the rust docs, I'll change it